src/main/java/ticTacToe/grid/GridFormatter.java

```diff
@@ -8,35 +8,42 @@
 public class GridFormatter {
 
     private static final String NEW_LINE = "\n";
-    private static final String LINE = "---------";
-    public static final String red_O =("\u001B[31mO\u001b[0m");
-    public static final String blue_X = ("\u001b[34mX\u001b[0m");
+    public static final String red_O = "\u001B[31mO\u001b[0m";
```

Comment:
"Global constants" in Java (i.e. `public static final` attributes) are usually written in all caps like the `NEW_LINE` that is already in this class.

Reply:
Also, if they are not referenced outside the class they are in, make them private.